### PR TITLE
Type definition for Lazar Laszlo's jsqrcode lib

### PR DESCRIPTION
**Pull request by lordazzi**
Type definition for Lazar Laszlo's jsqrcode lib.
This package was never on DefinitelyTyped.
Lazlo's is a javascript lib to read qrcode, it isn't an npm package and you can get his lib here: https://github.com/LazarSoft/jsqrcode

Thank you

### CODE REVIEW

**Comment by lordazzi** [comment]
I haven't created a test file for the definitions.
The travis is falling because there are no test file.
I belive that the correct name for the lib would be https://www.npmjs.com/package/@types/jsqrcode, according to the other libraries.

File: types/jsqrcode/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+    MIT License
```
Comment (mhegazy) [inline types/jsqrcode/LICENSE]: please do not include a separate licence files. all definition packages under DT are distributed under https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/LICENSE

File: types/jsqrcode/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Installation
```
Comment (mhegazy) [inline types/jsqrcode/README.md]: do not add a README either.

File: types/jsqrcode/package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (mhegazy) [inline types/jsqrcode/package.json]: do not add a package.json.

File: types/jsqrcode/index.d.ts [hunk 4]
```diff
@@ -0,0 +1,487 @@
+// Type definitions for Lazar Laszlo's jsqrcode
```
Comment (mhegazy) [inline types/jsqrcode/index.d.ts]: add a tslint.json, this should flag errors in your header.

**Review by mhegazy — CHANGES_REQUESTED** [review]
Please follow the template in https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package

File: types/jsqrcode/tsconfig.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "dom",
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```
Comment (mhegazy) [inline types/jsqrcode/tsconfig.json]: this should be true as well for new declarations.

**Comment by lordazzi** [comment]
I have fixed it all

**Comment by mhegazy** [comment]
thanks and welcome to @types :)

**Comment by lordazzi** [comment]
Thanks!

**Pull request merged**